modern/src/App.js

```diff
@@ -273,6 +284,40 @@ function Button({
   );
 }
 
+function Popupmenu({ id, node }) {
+  const children = node.commands.map(item => {
+    if (item.id === "seperator") {
+      return <li />;
+    }
+    return (
+      <li
+        key={item.id}
+        onClick={() => {
+          node.resolveCmdSelection(item.id);
+        }}
+      >
+        {item.name}
+      </li>
+    );
+  });
+  const { x, y } = mouseposition;
+  return (
+    <div
+      data-node-type="Popmenu"
+      data-node-id={id}
+      style={{
+        position: "absolute",
+        top: Number(y),
+        left: Number(x),
+        backgroundColor: "#000000",
+        color: "#FFFFFF",
```

Comment:
Maybe some `// TODO` comment here noting that we should come back to this.

Reply:
👍